package.json

```diff
@@ -173,14 +172,13 @@
     "es6-promise": "^4.2.8",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-config-carbon-base": "^1.0.1",
```

Comment:
I also had to remove `eslint-config-carbon-base` because it uses outdated dependencies.
But I have migrated [the small config](https://unpkg.com/browse/eslint-config-carbon-base@1.0.1/index.js).